### ISSUE TITLE
t.list is broken in 0.7.7

### ISSUE THREAD

**Issue by blairbodnar** [gcanti/tcomb-form#265]
I upgraded from 0.7.6 to 0.7.7 and t.list seems to have a problem. In the example below, the text inputs for the tags do not accept input from the user. 
```js
const Person = t.struct({
  name: t.String,
  tags: t.list(t.String) // broken?
});
```

Also, for the given Person value:
```js
const person = {
  name: 'John',
  tags: ['employee'] // will render an empty input
}
```
this will render one tag input with an empty value.

**Comment by gcanti** [comment]
Should be fixed in v0.7.8, thanks a lot @blairbodnar, sorry for the inconvenience

**Comment by blairbodnar** [comment]
Awesome! Wow that's fast :) Thanks!
